ui/Cargo.toml

```diff
@@ -4,6 +4,10 @@ name = "ui"
 version = "0.1.0"
 build = "build.rs"
 
+[features]
+default = ['pids-limit']
+pids-limit = []
```

Comment:
Maybe add a comment here along the lines of:

```toml
# This feature prevents forkbombs
```

To make it blatantly obvious to anyone disabling it what issues they may be opting into.